lib/WorseReflection/Bridge/Phpactor/DocblockParser/PHPStanParsedDocblock.php

```diff
@@ -0,0 +1,321 @@
+        $types = [];
+        foreach ($this->node->getTypeAliasTagValues() as $tag) {
+            $types[] = new DocBlockTypeAlias(
+                $tag->alias,
+                $this->typeConverter->convert($tag->type),
+            );
+        }
+
+        return new DocBlockTypeAliases($types);
+    }
+
+    public function methodType(string $methodName): Type
+    {
+        foreach ($this->node->getMethodTagValues() as $methodTag) {
+            if ($methodTag->methodName !== $methodName) {
+                continue;
+            }
+            $this->convertType($methodTag->returnType);
+        }
+
+        return TypeFactory::undefined();
+    }
+
+    public function inherits(): bool
+    {
+        return false;
+    }
+
+    public function vars(): DocBlockVars
+    {
+        $vars = [];
+        foreach ($this->node->getVarTagValues() as $varTag) {
+            assert($varTag instanceof VarTagValueNode);
+            $vars[] = new DocBlockVar(
+                ltrim($varTag->variableName, '$'),
+                $this->convertType($varTag->type),
+            );
+        }
+
+        return new DocBlockVars($vars);
+    }
+
+    public function params(): DocBlockParams
+    {
+        $params = [];
+        foreach ($this->node->getParamTagValues() as $paramTag) {
+            $params[] = new DocBlockParam(
+                ltrim($paramTag->parameterName, '$'),
+                $this->convertType($paramTag->type),
+            );
+        }
+
+        return new DocBlockParams($params);
+    }
+
+    public function parameterType(string $paramName): Type
+    {
+        $types = [];
+        foreach ($this->node->getParamTagValues() as $paramTag) {
+            assert($paramTag instanceof ParamTagValueNode);
+            if (ltrim($paramTag->parameterName, '$') !== $paramName) {
+                continue;
+            }
+            return $this->convertType($paramTag->type);
+        }
+
+        return TypeFactory::undefined();
+    }
+
+    public function propertyType(string $propertyName): Type
+    {
+        $types = [];
+        foreach ($this->node->getPropertyTagValues() as $propertyTag) {
+            if (ltrim($propertyTag->propertyName, '$') !== $propertyName) {
+                continue;
+            }
+            return $this->convertType($propertyTag->type);
+        }
+
+        return TypeFactory::undefined();
+    }
+
+    public function formatted(): string
+    {
+        return implode(
+            "\n",
+            array_map(
+                static fn (string $line) => preg_replace('{^\s+}', '', $line),
+                explode(
+                    "\n",
+                    $this->node->__toString()
+                )
+            )
+        );
+    }
+
+    public function returnType(): Type
+    {
+        foreach ($this->node->getReturnTagValues() as $tag) {
+            return $this->convertType($tag->type);
+        }
+
+        foreach ($this->node->getReturnTagValues('@psalm-return') as $tag) {
```

Comment:
This should be more like https://github.com/phpstan/phpstan-src/blob/eed9282f97246c03ec4374549b7f3e1712c48f06/src/PhpDoc/PhpDocNodeResolver.php#L444 to actually support different prefixes